### ISSUE TITLE
Line numbers

### ISSUE THREAD

**Issue by vitalyster** [CodeEditApp/CodeEditTextView#115]

**Comment by lukepistrol** [comment]
We probably won't hide the line numbers by default but having an option to turn them off seems like a good idea 👍

**Comment by ben-p-commits** [comment]
@lukepistrol - I'll take this one on.

**Comment by ben-p-commits** [comment]
Anecdotal- but I believe people who ditch line numbers are probably good with eschewing folding, review- etc.

**Comment by ben-p-commits** [comment]
<img width="228" alt="image" src="https://user-images.githubusercontent.com/5067237/214146817-bf85be60-07c4-42b0-88dc-35e89a8e5971.png">
then again...

xcode goes all the way

<img width="92" alt="image" src="https://user-images.githubusercontent.com/5067237/214146836-a5b27b06-d0b0-4862-af54-04b614231d1d.png">

**Comment by ben-p-commits** [comment]
@austincondiff,  perhaps we add the options as separate enhancements when the respective functionality is implemented?

Hard to say whether the RulerView is going to be the container for those futures like a code ribbon in the future, or if sibling views are needed for them.

**Comment by austincondiff** [comment]
I like that. If Xcode provides the ability to show/hide each thing individually, we should do the same.

**Comment by ben-p-commits** [comment]
So- do we wait for those features to be implemented? 😆

**Comment by ben-p-commits** [comment]
https://github.com/CodeEditApp/CodeEditTextView/pull/124

@austincondiff , @lukepistrol, PR in the pocket.

**Comment by austincondiff** [comment]
@ben-p-commits No, just making a comment that we should hide just the line numbers here and the others as they get implemented. 😉
